packages/server/src/controllers/index.controller.ts

```diff
@@ -14,7 +14,7 @@ export class IndexController extends Controller {
   }
   @Post('/webhook')
   async webhook(
-    @Header('Livepeer-Signature') livepeerSignature: string,
```

Comment:
I had lots of headaches with this. You sure it still works?

Reply:
No, it stopped validating signature on dev serve.